doc/common/glossary.md

```diff
@@ -79,7 +81,7 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
 - **Cloud Infrastructure Hardware Configuration:** a set of settings (Key:Value) that are applied/mapped to **Cloud Infrastructure** HW deployment.
 
-- **Cloud Infrastructure Hardware Profile:** defines the behaviour, capabilities, configuration, and metrics provided by a cloud infrastructure hardware layer.
+- **Cloud Infrastructure Hardware Profile:** defines the behaviour, capabilities, configuration, and metrics provided by a cloud infrastructure hardware layer resources available for the workloads.
```

Comment:
Why do we need to add this? The hardware profile defines that for any resources, also if they are unavailable.

Reply:
@ulikleber  is "... on resources for workloads" better?